code/_onclick/ai_onclick.dm

```diff
@@ -177,6 +177,11 @@
 // APC
 
 /obj/machinery/power/apc/AICtrlClick(mob/living/user) // turns off/on APCs.
+	if(aidisabled)
+		to_chat(user, "<span class='warning'>The AI wire is cut - Cannot toggle breaker.</span>")
```

Comment:
Just a grammar/style preference, but I would like something how it says on airlocks when the ai is disabled.
```suggestion
		to_chat(user, "<span class='warning'>Unable to interface: Connection refused.</span>")
```

Reply:
I copies template from Alt Clicking door text